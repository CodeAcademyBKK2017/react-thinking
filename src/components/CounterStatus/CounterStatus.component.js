import React ,{Component} from 'react';
import PropTypes from 'prop-types';
class CounterStatus extends Component {
  render () {
    return (
      <input type="text" value={this.props.value} />
    );
  }
}
CounterStatus.propTypes = {
  value: PropTypes.number.isRequired
};
CounterStatus.defaultProps = {
  value: 0
};
export default CounterStatus;