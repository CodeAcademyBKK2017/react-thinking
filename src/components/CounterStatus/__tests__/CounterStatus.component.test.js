import React from 'react';
import renderer from 'react-test-renderer';
import CounterStatus from '../CounterStatus.component';

describe('CounterStatus component', () => {
  it('CounterStatus: Snapshot test', () => {
    const snapshot = renderer.create(<CounterStatus value={5} />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});