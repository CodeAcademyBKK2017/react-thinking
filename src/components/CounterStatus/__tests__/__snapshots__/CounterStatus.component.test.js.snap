// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CounterStatus component CounterStatus: Snapshot test 1`] = `
<input
  className="Counter-box"
  type="text"
  value={5}
/>
`;
